program-runtime/src/instruction_processor.rs

```diff
@@ -342,6 +342,16 @@ impl InstructionProcessor {
         }
     }
 
+        /// Add a static entrypoint to intercept instructions before the dynamic loader.
```

Comment:
nits: `/// Remove ...`

Reply:
Yeah, this pr is super wip, but want feedback on the gist before spending time honing

Reply:
kk